app/controllers/admin/base_controller.rb

```diff
@@ -1,10 +1,10 @@
 class Admin::BaseController < ApplicationController
   before_filter :authenticate_admin
-  
+
   def index
-    @user = self.current_user
+    @user = current_user
```

Comment:
I guess it broke since moving `current_user` to a concern, this makes it work again.